translations/pt.po

```diff
@@ -617,22 +624,22 @@ msgstr ""
 #: src\locale.ts:66
 msgctxt "command-option"
 msgid "input-language"
-msgstr ""
+msgstr "idioma de entrada"
```

Comment:
I'll add a note to the translations documentation, but Discord arbitrarily enforces a rule for command names and options, in the spirit of the original English ones, prohibiting majuscule characters or spaces.

```suggestion
msgstr "idioma-de-entrada"
```